src/Vault/VaultClient.cs

```diff
@@ -111,7 +112,7 @@ internal Task DeleteVoid(string path, CancellationToken ct)
             return _httpClient.DeleteVoid(BuildVaultUri(path), _token, ct);
         }
 
-        private Uri BuildVaultUri(string path, NameValueCollection parameters = null)
+        private Uri BuildVaultUri(string path, DictionaryEntry[] parameters = null)

```

Comment:
This could probably be IEnumerable<KeyValuePairs<string, string>>

Reply:
yup, for some reason I was thinking that wasn't in 1.1, but it is..